src/liboslexec/shadingsys.cpp

```diff
@@ -1278,6 +1281,14 @@ ShadingSystemImpl::getattribute (string_view name, TypeDesc type,
     ATTR_DECODE ("stat:mem_inst_connections_current", long long, m_stat_mem_inst_connections.current());
     ATTR_DECODE ("stat:mem_inst_connections_peak", long long, m_stat_mem_inst_connections.peak());
 
+    if (name == "error_repeats") {
```

Comment:
Fun fact: Realized you are actually in getattribute, not setattribute here....

Reply:
Sorry, you are right. Fix is #883 and I will backport to all branches where I merged the error_repeats the first time.